Makefile.am

```diff
@@ -25,6 +25,7 @@ swupd_SOURCES = \
 	src/download.c \
 	src/extra_files.c \
 	src/filedesc.c \
+	src/fs_storage.c \
```

Comment:
Suggestion: Rename this to fs.c so we can move later all fs related functions (mkdir_p, is_populated_dir, copy_all, etc) from helper.c to fs.c.